nextflow_schema.json

```diff
@@ -504,6 +504,55 @@
                     "description": "Specify the -k parameter for BWA mem i.e. the  minimum seed length",
                     "help_text": "Configures the re-seeding used in BWA-MEM. Default is set to BWA default.\n\n> Modifies BWA-MEM parameter: `-r`",
                     "fa_icon": "fas fa-angle-double-left"
+                },
+                "mapping_bowtie2_alignmode": {
+                    "type": "string",
```

Comment:
Missing the `enum` (in schema builder, press cog and insert the `|` separated list)